coalib/settings/ConfigurationGathering.py

```diff
@@ -143,6 +143,7 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
 
     :param arg_list:    The list of command line arguments.
     :param log_printer: The LogPrinter object for logging.
+    :param arg_parser:  An ``ArgParser`` object.
```

Comment:
Should be `ArgumentParser` or fully qualified `argparse.ArgumentParser` to clearly know what is meant here. I know that just `ArgParse` is used several times in `coalib` doc strings... I will open an issue for that